odoo/fields.py

```diff
@@ -3363,9 +3363,10 @@ def _setup_definition_attrs(self):
             assert self.definition.count(".") == 1
             self.definition_record, self.definition_record_field = self.definition.rsplit('.', 1)
 
-            # make the field computed, and set its dependencies
-            self._depends = (self.definition_record, )
-            self.compute = self._compute
+            if not self.related:
```

Comment:
I am wondering if the issue is not so much that it's a related field, but more a computed field.
How about changing `if not self.related` into `if not self.compute`? (not sure)

Just tested the branch, and it seems to work fine.

Reply:
Actually, my fix is only for related field generated from `inherits`
Simple related and computed properties fields seems not carefully considered in 17.0 expecially when `read`
I will make a POC fix to check how much we can support them.